src/Native/Windows/clrcompression/zlib-intel/README.txt

```diff
@@ -0,0 +1,11 @@
+Zlib-Intel is an optimized version of Zlib developed by Intel to take advantage of modern CPU features. 
```

Comment:
Do we need to make any changes to the 3rd party notices file?
https://github.com/dotnet/corefx/blob/master/THIRD-PARTY-NOTICES#L65

cc: @richlander, @martinwoodward